stdlib/ext/file-ext.mc

```diff
@@ -105,6 +111,18 @@ utest
   else ("Error reading file","","","")
 with ("Hello", "Next string", "Final", "EOF") in
 
+-- Test reading x amount of characters from the file
+utest
+  match readOpen filename with Some rc then
+    let l1 = match readBytes rc 3 with Some s then s else "" in
```

Comment:
It seems slightly weird to default to "". 

Maybe `match readBytes rc 3 with Some s in s` 

This might be preferable since not being able to read bytes seems to be an error which defaulting to "" handles silently

Reply:
That's a good suggestion, I've added it!